cmake/Config.cmake.in

```diff
@@ -4,8 +4,8 @@
 #  @CMAKE_PROJECT_NAME@_INCLUDE_DIRS - Include directories for @CMAKE_PROJECT_NAME@
 #  @CMAKE_PROJECT_NAME@_LIBRARIES    - Libraries for @CMAKE_PROJECT_NAME@
 
-set(FLIRT_INCLUDE_DIRS @CMAKE_INSTALL_PREFIX@/include/flirtlib)
+set(FLIRT_INCLUDE_DIRS ${CMAKE_CURRENT_LIST_DIR}/../../../include/flirtlib)
 
-foreach(lib feature geometry gui sensors sensorstream utils)
```

Comment:
Note that I've dropped linking the GUI library by default, since it doesn't build if you don't have QT4. There are a dozen better ways to deal with this, but in the short term, I'm just eliminating it as we don't need it.